src/components/Tooltip.test.tsx

```diff
@@ -8,14 +8,21 @@ describe("Tooltip", () => {
     expect(resolveTooltip(false)).toBe(undefined);
     expect(resolveTooltip(true, null)).toBe(undefined);
 
-    // `disabled` param should always trump `tooltip` param
+    // `disabled` param should always trump `tooltip` and `readOnly` params
```

Comment:
**suggestion:** Looks like we aren't covering the case described by this comment? Maybe add a test for disabled = node, tooltip = anything, readOnly = node

Reply:
Definitely! Thanks for catching that.